sycl/test-e2e/ESIMD/slm_init_no_inline.cpp

```diff
@@ -27,6 +25,12 @@ int main() {
             << ", driver=" << D.get_info<info::device::driver_version>()
             << std::endl;
 
+  if (!isGPUDriverGE(Q, esimd_test::GPUDriverOS::LinuxAndWindows, "26516",
```

Comment:
@aelovikov-intel, do I recall correctly that we have a LIT feature to test against the driver version and report the test status correctly?

Reply:
Yes, since https://github.com/intel/llvm/pull/11427.

Reply:
@turinevgeny, I suggest using `REQUIRES-INTEL-DRIVER` instead this if-condition.